provision_scripts/install_pyart_notebooks.sh

```diff
@@ -4,9 +4,11 @@
 set -x
 # Get the Py-ART notebooks from GitHub
 cd ~
-#install git just in case
+#install git and zip/unzip just in case
 sudo apt-get install -qq git
-git clone https://github.com/openradar/pyart_short_course
-#get data
-cd pyart_short_course/data
-./get_data.sh
+sudo apt-get install -qq zip unzip
+git clone https://github.com/openradar/AMS-Short-Course-on-Open-Source-Radar-Software.git pyart_short_course
+
+# Optionally get data, add ~1 GB to the VM size
+#cd pyart_short_course/data
+#./provision_data.sh
```

Comment:
I guess it should be your call whether the data should be on board or not. 

Of course, adding another GB is something. I think for the ERAD course, missing data won't be an issue because there will not be much time (and expertise) to delve into the Py-ART notebooks. Thus, it'll be enough for the participants to just browse through the Py-ART notebooks and just inspect the cell output. For the wradlib notebooks, I consider it important that the cell output is cleared in all notebooks, but for Py-ART, the opposite might be better this time...

Anyway, I leave this up to you.